lib/both.js

```diff
@@ -54,25 +54,46 @@ DocHead = {
     }
   },
   _buildTag(metaInfo, type) {
-    let props = "";
-    for (let key in metaInfo) {
-      props += `${key}="${metaInfo[key]}" `;
+    let meta;
+    if (Meteor.isClient) {
+      meta = document.createElement(type);
```

Comment:
You don't need to create a html node here. You can simply append string to the DOM using `insertAdjacentHTML`. See: http://stackoverflow.com/a/7327125

With that, you don't need to touch this code.